omnisend/class-omnisend-core-bootstrap.php

```diff
@@ -55,11 +57,20 @@ public static function load(): void {
 
 		if ( ! self::is_omnisend_woocommerce_plugin_active() || ! self::is_omnisend_woocommerce_plugin_connected() ) {
 			add_action( 'wp_footer', 'Omnisend\Internal\Snippet::add' );
-		}
 
-		add_action( OMNISEND_CORE_CRON_SYNC_CONTACT, 'Omnisend\Internal\Sync::sync_contacts' );
-		add_action( 'user_register', 'Omnisend\Internal\Sync::hook_user_register' );
-		add_action( 'profile_update', 'Omnisend\Internal\Sync::hook_profile_update' );
+			add_action( 'user_register', 'Omnisend\Internal\Sync::hook_identify_user_by_id' );
+			add_action(
+				'wp_login',
+				function ( $user_login, $user ) {
+					Omnisend\Internal\Sync::hook_identify_user_by_id( $user->ID );
+				},
+				10,
```

Comment:
Please remove those additional params. 10 is default, 2 is not correct.

Reply:
2 should be correct https://developer.wordpress.org/reference/hooks/wp_login/

Reply:
This is number of argument callback (your function) accepts. `identify_user_by_id` accepts only one argument. So default values are ok, and those 10, 2 can be deleted.

Reply:
My callback function is an anonymous function that accepts two arguments 
`function ( $user_login, $user ) {`. 

I believe I tried creating `add_action` without these arguments and it failed.

Reply:
ohh 🤦 you right 👍